lib/modules/manager/tekton/__fixtures__/multi-doc.yaml

```diff
@@ -0,0 +1,146 @@
+---
```

Comment:
i think you can remove some of the fixtures docs without loosing coverage. 

all of them are docker images, so you should only test the doc structure and not every permutation of a docker image reference

Reply:
Sure thing. Removed some cases.